pom.xml

```diff
@@ -0,0 +1,51 @@
+<project>
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.cloud.tools</groupId>
+  <artifactId>gcp-java-tools-lib</artifactId>
+  <version>0.2-SNAPSHOT</version>
+
+  <name>Google Cloud Java management client library</name>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <appengine.target.version>1.9.32</appengine.target.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-tools-sdk</artifactId>
+      <version>${appengine.target.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+      <version>19.0</version>
```

Comment:
I'm not sure what you mean about porting the libraries.

The issue is that IJ bundles guava as part of the IJ SDK, so you could conceivably have classloader issues if we depend on this from our IJ plugin.